pkg/maas/maintenance/config.go

```diff
@@ -59,9 +60,10 @@ type VECConfig struct {
 // DefaultHMCConfig returns conservative defaults for HMC operation.
 func DefaultHMCConfig() HMCConfig {
 	return HMCConfig{
-		PerWLCMoveTimeout:  20 * time.Minute,
-		PerHostWaveTimeout: 60 * time.Minute,
-		Policy:             ForceHalt,
+		PerWLCMoveTimeout:       20 * time.Minute,
+		PerHostWaveTimeout:      60 * time.Minute,
+		EvacuationCheckInterval: 30 * time.Second,
+		Policy:                  ForceHalt,
```

Comment:
While DefaultHMCConfig is part of design, lets not use it in flow. As we are prioritizing WLC lifecycle over HCP cluster upgrade flow.